### PR TITLE
Jetpack

### PR DESCRIPTION
**Pull request by gillespieza**
Added the id of the sidebar "footerfull" to the infinite-scroll argument list, so that if there are any widgets in the footer, the default Infinite Scroll behaviour would change to on-click instead.

Added a scss file to make the default layout of the Infinite Scroll button and footer match Bootstrap 4.

### CODE REVIEW

File: sass/theme.scss [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 
 //Optional files - If you dont use the corresponding scripts/fonts comment em out
 @import "assets/font-awesome"; // <------- Font Awesome Icon font
 @import "assets/underscores"; // <------- Underscores media styles
 
 // Any additional imported files //
-@import "theme/theme";  // <--------- That's where you can add your own design. Thats your part!
+@import "theme/theme"; // <--------- That's where you can add your own design. Thats your part!
 @import "theme/contact-form7"; // Contact Form 7 - Bootstrap 4 support
+@import "theme/infinite-scroll"; // Infinite Scroll from JetPack.
```
Comment (IanDelMar) [inline sass/theme.scss]: File `sass/theme/_infinite-scroll.scss` is missing. The import of non-existing files will throw an error on compilation.

Comment (gillespieza) [reply sass/theme.scss]: Ooops! I've added the missing file.

**Comment by danemorgan** [comment]
As it's been more than a year and a half, and we have conflicts here, I wonder if closing this pull request and making a fresh go of it starting from the current code would be a better approach?

**Comment by danemorgan** [comment]
Closing as stale.

**Pull request closed without merging**